include/xtensor/xscalar.hpp

```diff
@@ -218,6 +219,12 @@ namespace xt
     }
 
     template <class T>
+    inline auto xscalar<T>::operator[](const xindex& idx) const noexcept -> const_reference
```

Comment:
You can comment or remove the parameter name to avoid the annoying warning "parameter not used"